tree-items.txt

```diff
@@ -506,14 +506,17 @@ xfs/ext4). In BTRFS a file extent can be in one of 3 states:
 
 * BTRFS_FILE_EXTENT_INLINE - an extent item who can hold all of the actual file
 data in its body is considered an inline extent. In such cases data is read
-directly from the body of the extent.
+directly from the body of the extent. The current default value is 2048 bytes,
+but it can be changed with max_inline mount option. The max_inline value is then
+calculated with min(max_inline_value, sectorsize).
```

Comment:
That min(...)  is going too much into implementation detail. I think it will be better if you simply stated that the maximum size of inline data can be up to 1 filesystem block (sectorsize in this context is not a block disk sector size, which is usually 512 bytes, but a filesystem sector which is the btrfs' way of referring to filesystem block size).

Reply:
yes, I tried to use sectorsize here because it can be changed by the --sectorsize argument of mkfs.btrfs, so that's why I used sector size instead of page size.